trie/trieStorageManager_test.go

```diff
@@ -85,13 +89,63 @@ func TestTrieCheckpoint(t *testing.T) {
 
 	dirtyHashes := trie.GetDirtyHashes(tr)
 
+	errChan := make(chan error, 1)
 	trieStorage.AddDirtyCheckpointHashes(rootHash, dirtyHashes)
-	trieStorage.SetCheckpoint(rootHash, []byte{}, nil, &trieMock.MockStatistics{})
+	trieStorage.SetCheckpoint(rootHash, []byte{}, nil, errChan, &trieMock.MockStatistics{})
 	trie.WaitForOperationToComplete(trieStorage)
 
 	val, err = trieStorage.GetFromCheckpoint(rootHash)
 	assert.Nil(t, err)
 	assert.NotNil(t, val)
+	assert.Equal(t, 0, len(errChan))
+}
+
+func TestTrieStorageManager_SetCheckpointEmptyErrorChan(t *testing.T) {
+	t.Parallel()
+
+	args := getNewTrieStorageManagerArgs()
+	ts, _ := trie.NewTrieStorageManager(args)
+	_ = ts.Close()
```

Comment:
Can we remove this line from this test?
Otherwise, if someone changes the code to not check for nil channel, this test would still pass (wrong behavior), because `TrieStorageManager` is closed

Reply:
right, moved the `_ = ts.Close()` as the last instruction in the test to free the resources involved